lib/slack/real_time/socket.rb

```diff
@@ -59,6 +64,14 @@ def restart_async(_client)
         raise NotImplementedError, "Expected #{self.class} to implement #{__method__}."
       end
 
+      def time_since_last_message
+        current_time - @last_message_at
+      end
+
+      def current_time
+        Time.now
```

Comment:
`Time.now` is not monotonic.

Reply:
https://blog.dnsimple.com/2018/03/elapsed-time-with-ruby-the-right-way/

Running to work to fix all the instances of `Time.now` that I have all over the place :)

Reply:
Haha, yeah, it's a common problem :p

Reply:
Solid feedback! Thanks for the link dblock! I'll change that.